common/handle-rejection.ts

```diff
@@ -44,11 +44,20 @@ export async function catchAll<T>(promise: PromiseLike<T>|
     let message: string;
     if (e instanceof AggregateError) {
       message = e.message;
+      for (const stackLine of e.stack.split(/\n/)) {
+        task.debug(stackLine);
```

Comment:
Is there a way to factor out the splitting of line logic?

Reply:
Do you want a separate variable that is the array of split lines, or are you concerned about the duplication of code in the `Error` and `AggregateError` blocks?